src/__stories__/stacking-group-story.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import {Box, ResponsiveLayout} from '..';
+import StackingGroup from '../stacking-group';
+import Image from '../image';
+import Avatar from '../avatar';
+import img from './images/avatar.jpg';
+
+export default {
+    title: 'Components/StackingGroup',
+    parameters: {
+        fullScreen: true,
+    },
+};
+
+type Args = {
+    stacked?: boolean;
+    type: 'circle' | 'square';
+    size: number;
+    maxItems?: number;
+    children?: React.ReactNode;
+    itemsToInclude: number;
+    inverse: boolean;
+};
+
+export const Default: StoryComponent<Args> = ({
+    stacked,
+    size,
+    type,
+    maxItems,
+    itemsToInclude,
+    inverse = false,
+}) => {
+    return (
+        <div data-testid="stacking-group" style={{width: 'max-content'}}>
```

Comment:
you don't need this div, and why do you add the width: 'max-content' ? it's not needed

If you want the data-testid for the screenshot test, you can use the `dataAttributes` prop in ResponsiveLayout or Box if you want:

```ts
<ResponsiveLayout isInverse={inverse} dataAttributes={{testid: 'stacking-group'}}>
```